tools/src/test/java/org/apache/kafka/tools/AbstractResetIntegrationTest.java

```diff
@@ -306,17 +304,17 @@ private void testReprocessingFromScratchAfterResetWithIntermediateUserTopic(fina
         final List<KeyValue<Long, Long>> resultRerun2 = IntegrationTestUtils.waitUntilMinKeyValueRecordsReceived(resultConsumerConfig, OUTPUT_TOPIC_2_RERUN, 40);
         streams.close();
 
-        assertThat(resultRerun, equalTo(result));
-        assertThat(resultRerun2, equalTo(result2));
+        assertEquals(result, resultRerun);
+        assertEquals(result2, resultRerun2);
 
         if (!useRepartitioned) {
             final Properties props = TestUtils.consumerConfig(cluster.bootstrapServers(), appID + "-result-consumer", LongDeserializer.class, StringDeserializer.class, commonClientConfig);
             final List<KeyValue<Long, String>> resultIntermediate = IntegrationTestUtils.waitUntilMinKeyValueRecordsReceived(props, INTERMEDIATE_USER_TOPIC, 21);
 
             for (int i = 0; i < 10; i++) {
-                assertThat(resultIntermediate.get(i), equalTo(resultIntermediate.get(i + 11)));
+                assertEquals(resultIntermediate.get(i), resultIntermediate.get(i + 11));
```

Comment:
ditto This `assertEquals` statement has the expected and actual values in the wrong order.

Reply:
Thanks, I missed those two assertions.